content/implement/client.md

```diff
@@ -0,0 +1,209 @@
++++
+date = "2016-01-30T11:01:06-05:00"
+title = "Client"
+weight = 9
+
+[menu.main]
+name = "Client"
+identifier = "implement client"
+parent = "implement"
++++
+
+Once the API dsl is used to define types and resources, goagen automatically generates a
+native go client, and optionally a javascript client based on the [axios](ihttps://github.com/mzabriskie/axios) 
+javascript package. 
+
+## Native Go Client
+
+In order to generate the go client `goagen client -d [design_path]` must be run. It is
+generates automatically in the bootstrap command as well. 
+
+The native go client makes a few different things to make consuming the API easier...
+
+#### A Payload struct (if applicable)
+
+If the endpoint you are consuming requires a payload, then it will auto generate a payload
```

Comment:
Please use the present stence:

The generated code contains a payload struct for each endpoint that requires a payload. The struct follows the definition provided in the API design.